sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,14 +56,8 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit c53953ae492587698d5adbab8ffee254d97b6a4e
-  # Merge: 9f88cf88 66d52ace
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Wed Jan 10 14:50:23 2024 +0000
-  #     Merge pull request #1170 from jchlanda/jakub/hip_custom_dirs
-  #     [HIP] Allow custom location of ROCm components
-  set(UNIFIED_RUNTIME_TAG c53953ae492587698d5adbab8ffee254d97b6a4e)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/bensuo/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG maxime/imm-cmd-list-support)
```

Comment:
Need to update target UR!

Reply:
Yes, this will be updated as soon as the UR PR is merged to the main branch.